### DISCUSSION TITLE
Feature Request: Notification for Device Limit in MeshCentral

### DISCUSSION THREAD

**Discussion by alanpereiracoelho** [Ylianst/MeshCentral#5651] [Issues]
Dear MeshCentral Community,

I am writing to share an observation and a potential improvement regarding the use of MeshCentral in managing devices. Following the guidance from a community member, si458, I have been configuring my server, including setting up the MaxDevices limit. The feature worked as expected, but I noticed that there was no notification from either the agent or the MeshCentral platform when the limit of added devices was reached.

I believe it would be highly beneficial for users to have a notification system in place to alert them when they are approaching or have reached the maximum number of devices allowed. Additionally, it would be helpful to have a visible counter somewhere in the user interface showing the current number of added devices and the total allowable devices. This enhancement could prevent confusion or the misconception of malfunction, especially in cases where users might not be aware of the device limit.

I hope this suggestion resonates with the community and the developers, and I look forward to any thoughts or feedback on this proposal.

Thank you for your time and for the continuous improvements to the MeshCentral platform.

Best regards,
Alan

**Comment by si458** [comment]
moved to an enhancement issue #5652
